charts/kserve-crd/templates/serving.kserve.io_inferencegraphs.yaml

```diff
@@ -562,6 +562,22 @@ spec:
               timeout:
                 format: int64
                 type: integer
+              tolerations:
+                items:
+                  properties:
+                    effect:
+                      type: string
+                    key:
+                      type: string
+                    operator:
+                      type: string
+                    tolerationSeconds:
+                      format: int64
+                      type: integer
+                    value:
+                      type: string
+                  type: object
+                type: array
```

Comment:
PR is still not finished, right?

As I see most changes are only applying tolerations.

Reply:
Ah! The charts CRD generation is not automated. I manually copied the CRD. I forgot to include the latest changes.